spec/services/forest_liana/value_stat_getter_spec.rb

```diff
@@ -0,0 +1,96 @@
+module ForestLiana
```

Comment:
[MISC] This test has only been moved and re-written to avoid ghost issues on the CI with scopes